testutils/ptest2/conf/deployed/master-mr2.properties

```diff
@@ -90,7 +90,7 @@ unitTests.module.hcatalog.core=hcatalog.core
 ut.hcatalog.core.batchSize=9
 
 
-qFileTests = clientPositive miniMr clientNegative miniMrNegative hbasePositive miniTez spark miniLlap miniLlapLocal encryptedCli miniSparkOnYarn
+qFileTests = clientPositive miniMr clientNegative miniMrNegative hbasePositive miniTez spark miniLlap miniLlapLocal miniLlapExtDB encryptedCli miniSparkOnYarn 
```

Comment:
I created a new qFileTest named miniLlapExtDB. I thought this is where we configure our test driver so that it can deploy on Jenkins? If not, I will revert the changes in this file